book/website/collaboration/stakeholder-engagement.md

```diff
@@ -0,0 +1,142 @@
+(cl-stakeholder-tools)=
+
+# Facilitating Stakeholder Engagement
+
+When working with stakeholders beyond your own team, there are many tools and best practices for improving the quality of engagement through activities like mapping relevant project stakeholders, organising stakeholder workshops, establishing shared language and understanding, and creating the foundation for effective collaboration.
+
+This chapter will share guidance and resources for facilitating different forms of stakeholder engagement for data science collaborations drawing upon expertise from the Turing Research Application Management (RAM) team.
+
+(cl-stakeholder-mapping)=
+## Stakeholder mapping
+### Key question: Who is involved in a collaboration, either as a user, impacted group, or as part of the community of practice?
+
+The goal of stakeholder mapping is to centralise information on the people and organisations involved in a collaboration in one place. 
```

Comment:
Centralise feels like the wrong word (reads as source of truth, which I really don't like for stakeholder mapping). Not sure on a better word though. Maybe this...
```suggestion
The goal of stakeholder mapping is to understand the people and organisations involved in a collaboration, and collect all this information in one place. 
```